easy-cluster/easy-cluster.sh

```diff
@@ -667,12 +672,12 @@ cat <<EOT > job.json
     "outputDirectories": [
       {
         "id": "MODEL",
-        "pathPrefix": "$AFS_DIRECTORY",
+        "pathPrefix": "$AFS_DIRECTORY/horovod/model",
```

Comment:
Why use `/model` here? The pathSuffix will add `models`. Right now, it would be: `$AFS_DIRECTORY/horovod/model/models`....is that what we want or just `AFS_DIRECTORY/horovod/model`?

Reply:
good question, I believe is a discrepancy between versions. Whatever you consider better, I just copied your files

Reply:
id just do `"pathPrefix": "$AFS_DIRECTORY/horovod` and keep the suffix as is